### ISSUE TITLE
[Accessibility][Info and Relationship - Property Pane_Ask a question (Bot Asks)]: Incorrect information is announced by the screen reader at max turn count spin button.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2116]
**User Experience** 
User who rely on screen reader will get confused if incorrect information is announced by the screen reader.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to 'Max turn count' spin button.
6. Observe the issue.

**Actual:** 
On getting the focus, narrator announces 0 and 100 as maximum and minimum value of the spin button but the user will able to select other value's.
Ex- User will able to select -1 and 999999 without any error.

**Expected:**
The value should be restricted between 0 to 100 as announced by the narrator. As well as whenever user selects value different from the announced series the narrator should announce any error message.

**Note:**
Same issue observed with every property pane which have spin button such as 'Connect to QnA Knowledgebase'.

**MAS Reference:** [MAS1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment:**
[incorrect info is announced about max turn count button.zip](https://github.com/microsoft/BotFramework-Composer/files/4262053/Usable.incorrect.info.is.announced.about.max.turn.count.button.zip)
![Min Max value](https://user-images.githubusercontent.com/60913131/75661191-e084c680-5c92-11ea-8484-dc841592b33d.png)

**Comment by beyackle** [comment]
This seems to not apply any more. The numeric input is just a box, not a spinner, and has no minimum or maximum values.

**Comment by corinagum** [comment]
@ashish315 @awalia13 could we get confirmation? Thank you!

**Comment by ashish315** [comment]
@corinagum As we checked on the 'Master Bot Composer', still have same button and issue still repro their. Can you please confirm fixes deployed on which build. I downloaded the 'Master' from 'https://github.com/microsoft/BotFramework-Composer/tree/master'.

**Comment by corinagum** [comment]
![image](https://user-images.githubusercontent.com/14900841/78826786-f7f05700-7996-11ea-96ed-2600a5da663c.png)
![image](https://user-images.githubusercontent.com/14900841/78826819-050d4600-7997-11ea-8c32-307deb9d124d.png)


I can confirm that behavior is sitll incorrect. It looks like the input is missing the (non-aria) attributes `min` and `max`. Keeping this in the To do list.

**Comment by beyackle** [comment]
Yeah, the adaptive-form change turned these back into spinners. I'll handle fixing this.

**Comment by beyackle** [comment]
So, there's a problem with SpinButton; it doesn't know how not to have min/max props. They're technically optional, but they default to 0 and 100, which is what we're seeing here. I've let the Fluent UI team know about this issue.

**Comment by beyackle** [comment]
https://github.com/microsoft/fluentui/issues/11358 - the issue on Fabric/Fluent's own tracker

**Comment by cwhitten** [comment]
@ashish315 can you please close this as a FluentUI bug?

**Comment by ashish315** [comment]
As it is external bug. So, HCL is closing the issue.
